### ISSUE TITLE
process_request: clean PTR & KEY records under _signal subdomain after sucessful KEY registration.

### ISSUE THREAD

**Issue by adam-burns** [NetworkCommons/sig0namectl#36]
To all front ends to determine whether KEY registration policy is successful or not, once a key registration request has been processed, whether the KEY passed registration tests or not:

The PTR record which points to the KEY request:

```
_signal.${ZONE} IN PTR ${requested_label}._signal.${ZONE}.
```
And the requested KEY under _signal:
```
${requested_label}._signal.${ZONE}. IN KEY ${KEY_RDATA}

are both deleted upon the script processing each request.

**Comment by adam-burns** [comment]
Merged in 4cc8ac8

**Comment by adam-burns** [comment]
For the domains zenr.io & beta.freifunk.net, the process_requests script now now run every 5 seconds.
